### ISSUE TITLE
IE 11 Issues in Tabs

### ISSUE THREAD

**Issue by owenthomasclark** [excid3/tailwindcss-stimulus-components#28]
Appreciate you're probably not too concerned about IE11 (understandably), but thought I'd share something that I came across as I unfortunately have to at the moment. Fingers crossed not for long!

It took me quite a while but I found the bit causing me issues was:

```javascript
panel.classList.toggle('hidden', index != this.index)
```

I'm using Rails 6, with Babel 7 and the Stimulus polyfill. I'm not actually using this as a plugin library but used the code as inspiration. Maybe there's a smarter way to get around this issue, maybe it's not an issue at all. But I purely switched this to a standard IF statement such as the below:

```javascript
if (index === this.index) {
 panel.classList.remove('hidden')
} else {
 panel.classList.add('hidden')
}
```

I barely touch JS these days so probably a much cleaner way of doing it, and maybe not an issue, i'm most likely doing something daft.

**Comment by excid3** [comment]
Good find @owenthomasclark 👍 

Looks like it's the second argument that's not supported. https://til.hashrocket.com/posts/mnm3du0kja-classlisttoggle-2nd-arg-doesnt-work-in-ie11

I'm fine with switching this to a regular if statement and we can just leave a comment in referencing this issue. It's not a big change, so it seems reasonable to do for some IE11 support which is a win I think.

Did you want to make a PR?

**Comment by excid3** [comment]
Fixed by #29
